ca/certificate-authority.go

```diff
@@ -169,6 +193,18 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		NotAfter:   issuer.NotAfter,
 	}
 
+	// TODO(rlb@ipv.sx): Add a stop channel to shut down the CA gracefully
```

Comment:
This is quite easy to implement it, instead of checking in a TODO can you add it so it isn't forgotten.